mlrun/datastore/sources.py

```diff
@@ -412,6 +413,51 @@ def to_dataframe(
             **reader_args,
         )
 
+    def _get_spark_additional_filters(self):
+        if not self.additional_filters:
+            return None
+        none_values = [None, np.nan, np.NaN]
+        additional_filters = deepcopy(self.additional_filters)
+        is_first = True
+        spark_filter = ""
+        for filter_tuple in additional_filters:
+            if not filter_tuple:
+                continue
+            if not is_first:
+                spark_filter += " AND "
+            else:
+                is_first = False
+            col_name, op, value = filter_tuple
+            if isinstance(value, (str, datetime)):
+                value = rf"'{value}'"
+            if isinstance(value, (list, tuple, set, np.ndarray)) and (
+                "in" in op.lower()
```

Comment:
Maybe clearer and safer to make this more specific. i.e. `op.lower() in ["in", "not in"]`.